packages/swirl-components/src/components/swirl-date-picker/swirl-date-picker.tsx

```diff
@@ -9,9 +9,20 @@ import {
 } from "@stencil/core";
 import { WcDatepickerCustomEvent } from "wc-datepicker/dist/types/components";
 import { WCDatepickerLabels } from "wc-datepicker/dist/types/components/wc-datepicker/wc-datepicker";
+import { getISODateString, removeTimezoneOffset } from "../../utils";
 
 import "wc-datepicker";
-import { getISODateString, removeTimezoneOffset } from "../../utils";
+
+interface WeekInfo {
+  firstDay: number;
+  weekendStart: number;
+  weekendEnd: number;
+}
```

Comment:
Don't think the object will look like that:
![image](https://github.com/user-attachments/assets/be5f4f76-02f5-4244-b5d5-20fc9778fd14)

Maybe just keep the `firstDay` in it and add a comment above for the source? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getWeekInfo

Reply:
Haha, the one time I trust ChatGPT. 🫣 Should have known better. Done.